app/javascript/react/src/components/App.js

```diff
@@ -1,18 +1,19 @@
 import React from 'react';
 import { Route, IndexRoute, Router, browserHistory } from 'react-router';
-import BookIndexContainer from './containers/BookIndexContainer';
-import BookShowContainer from './containers/BookShowContainer'
+import BookIndexContainer from '../containers/BookIndexContainer';
+import BookShowContainer from '../containers/BookShowContainer';
+import HomeIndexContainer from '../containers/HomeIndexContainer';
 
 const App = props => {
   return(
     <div>
+      <h1>Welcome to our site!</h1>
       <Router history={browserHistory}>
-
+        <Route path="/" component={HomeIndexContainer}/>
         <Route path='/books' component={BookIndexContainer}/>
+        <Route path='/books/new'/>
```

Comment:
Please either make a placeholder container for this route - or remove it for now